### PR TITLE
Follow-up edits to PR #1723

### PR DESCRIPTION
**Pull request by tpoitras**
Nothing major. Just some edits related to #1723 

If there aren't any problems, some one can hit merge.

@adellape @vikram-redhat @tnguyen-rh @bfallonf @ahardin-rh 

Related: https://bugzilla.redhat.com/show_bug.cgi?id=1293578#c36

### CODE REVIEW

**Comment by nak3** [comment]
Thank you @tpoitras and @vikram-redhat it looks great to me.

File: install_config/upgrading/manual_upgrades.adoc [hunk 1]
```diff
@@ -922,14 +922,16 @@ process].
 [[manual-step-ose-3-1-1]]
 === OpenShift Enterprise 3.1.1
 
-There was an issue with OpenShift Enterprise 3.1.1 which involved hosts where their 
-hostnames resolves to an IP address that's not local to the host would run into
-problems with liveness and readiness probes on newly created HAProxy Routers. This was resolved in
+There was an issue with OpenShift Enterprise 3.1.1, where hosts with hostnames
```
Comment (adellape) [inline install_config/upgrading/manual_upgrades.adoc]: s/hostnames/host names/

Comment (adellape) [inline install_config/upgrading/manual_upgrades.adoc]: I don't think the comma is needed?

File: install_config/upgrading/manual_upgrades.adoc [hunk 2]
```diff
@@ -922,14 +922,16 @@ process].
 [[manual-step-ose-3-1-1]]
 === OpenShift Enterprise 3.1.1
 
-There was an issue with OpenShift Enterprise 3.1.1 which involved hosts where their 
-hostnames resolves to an IP address that's not local to the host would run into
-problems with liveness and readiness probes on newly created HAProxy Routers. This was resolved in
+There was an issue with OpenShift Enterprise 3.1.1, where hosts with hostnames 
+that resolved to IP addresses that were not local to the host would run into
+problems with liveness and readiness probes on newly-created HAProxy Routers.
```
Comment (adellape) [inline install_config/upgrading/manual_upgrades.adoc]: s/Routers/routers/

File: install_config/upgrading/manual_upgrades.adoc [hunk 3]
```diff
@@ -922,14 +922,16 @@ process].
...
-There was an issue with OpenShift Enterprise 3.1.1 which involved hosts where their 
-hostnames resolves to an IP address that's not local to the host would run into
-problems with liveness and readiness probes on newly created HAProxy Routers. This was resolved in
+There was an issue with OpenShift Enterprise 3.1.1, where hosts with hostnames 
+that resolved to IP addresses that were not local to the host would run into
+problems with liveness and readiness probes on newly-created HAProxy Routers. 
+This was resolved in
 https://access.redhat.com/errata/product/290/ver=3.1/rhel---7/x86_64/RHBA-2016:0293[RHBA-2016:0293]
-by configuring the probes to use 'localhost' as the hostname for pods utilizing hostPorts.
+by configuring the probes to use 'localhost' as the hostname for pods utilizing
```
Comment (adellape) [inline install_config/upgrading/manual_upgrades.adoc]: `*localhost*`

File: install_config/upgrading/manual_upgrades.adoc [hunk 4]
```diff
@@ -922,14 +922,16 @@ process].
...
-hostnames resolves to an IP address that's not local to the host would run into
-problems with liveness and readiness probes on newly created HAProxy Routers. This was resolved in
+There was an issue with OpenShift Enterprise 3.1.1, where hosts with hostnames 
+that resolved to IP addresses that were not local to the host would run into
+problems with liveness and readiness probes on newly-created HAProxy Routers. 
+This was resolved in
 https://access.redhat.com/errata/product/290/ver=3.1/rhel---7/x86_64/RHBA-2016:0293[RHBA-2016:0293]
-by configuring the probes to use 'localhost' as the hostname for pods utilizing hostPorts.
+by configuring the probes to use 'localhost' as the hostname for pods utilizing 
+hostPorts.
```
Comment (adellape) [inline install_config/upgrading/manual_upgrades.adoc]:
```
pods with `*hostPort*` values.
```

File: install_config/upgrading/manual_upgrades.adoc [hunk 5]
```diff
@@ -922,14 +922,16 @@ process].
...
 
-If you created a router under the affect version you should add 'host: localhost'
-if your liveness or readiness probes unexpectedly failing for your router.
+If you created a router under the affected version, and your liveness or 
+readiness probes unexpectedly fail for your router, then add 'host: localhost':
```
Comment (adellape) [inline install_config/upgrading/manual_upgrades.adoc]:
```
*host: localhost*
```
or
```
`host: localhost`
```
Our guidelines aren't good about markup for something like this. :neutral_face:

File: release_notes/ose_3_1_release_notes.adoc [hunk 6]
```diff
@@ -549,14 +549,16 @@ properly cached and used for volume management.
...
+This was resolved in
 https://access.redhat.com/errata/product/290/ver=3.1/rhel---7/x86_64/RHBA-2016:0293[RHBA-2016:0293]
-by configuring the probes to use 'localhost' as the hostname for pods utilizing hostPorts.
+by configuring the probes to use 'localhost' as the hostname for pods utilizing 
+hostPorts.
 
-If you created a router under the affect version you should add 'host: localhost'
-if your liveness or readiness probes unexpectedly failing for your router.
+If you created a router under the affected version, and your liveness or 
+readiness probes unexpectedly fail for your router, then add 'host: localhost':
```
Comment (adellape) [inline release_notes/ose_3_1_release_notes.adoc]: See comments in `manual_upgrades.adoc`,  as they're all the same for this topic too.

**Comment by tpoitras** [comment]
Thanks @adellape 

Updated as per your peer review notes.

Should be fine to merge now. :melon:

**Comment by adellape** [comment]
@tpoitras Veryfine. :tangerine:

**Comment by tpoitras** [comment]
[rev_history]
|link:../install_config/upgrading/manual_upgrades.html[Upgrading -> Manual Upgrades]

|Changed a known issue to a fix regarding liveness and readiness probes.

**Comment by vikram-redhat** [comment]
@tpoitras - Missing a % at the end. Added now.

**Pull request merged**